resources/scss/views/ItemList/Components/item-color-tiles.scss

```diff
@@ -0,0 +1,47 @@
+.fl-item-color-tiles-container {
+    .fl-item-color-tiles-list {
+        padding-left: 0;
+
+        .fl-item-color-tiles-list-item {
+            margin: 10px 10px 10px 0;
+            display: inline-block;
+
+            .fl-color-tile-label {
+                border: 1px solid $gray-lighter2;
+                position: relative;
+                cursor: pointer;
+
+                svg {
+                    width: 24px;
+                    height: 24px;
+                    margin: 3px;
+
+                    path {
+                        stroke: $black;
+                        fill: $white;
+                    }
+                }
+                .fl-color-tile-background {
+                    width: 30px;
+                    height: 30px;
+                    text-align: center;
+
+                    &:hover {
+                        border: 1px solid #ddd;
```

Comment:
I have overseen this in the last review. Please use `$custom-file-border-color`.

Reply:
I would recommend to create a new Findologic plugin variables file and define this variable with name in this plugin context, something like $color-tiles-border-color.

Reply:
We won't need this right now.